src/pages/exception/404/index.tsx

```diff
@@ -0,0 +1,25 @@
+import Link from 'umi/link';
+import { Result, Button } from 'antd';
+import React from 'react';
+import { formatMessage } from 'umi-plugin-react/locale';
+
+export default () => (
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=4583897)